api/storage/objectstore.py

```diff
@@ -45,8 +45,8 @@ def delete(self, document: Document):
         container_path = ObjectStore.get_container_path(document.type)
         try:
             connection.delete_object(container_path, document.filename)
-        except ClientException as e:
-            logger.info(f"Failed to delete object: {e}")
+        except ClientException:
+            logger.info(f"Failed to delete object for document id {document.id}")
 
         logger.info("Done deleting file from objectstore")
```

Comment:
Moet deze log niet in de `else:` van het try except block?

Reply:
Goede opmerking. Dat zou inderdaad netter zijn, echter voor nu wil ik geen changes meer doen.